docs/content/preview/benchmark/sysbench-ysql.md

```diff
@@ -111,17 +118,15 @@ $ sysbench <workload>               \
 
 ## Expected results
 
-The following results are for a 3-node cluster with each node on a c5.4xlarge AWS instance (16 cores, 32 GB of RAM, and 2 EBS volumes), all in the same AZ with the client VM running in the same AZ.
+The following results are for a 3-node cluster, with each node running on a c5.2xlarge AWS instance (8 cores, 16 GiB of RAM), all in the same AZ, with a replication factor of 3 and TLS enabled.
 
 ### 10 tables each with 100k rows
 
-|       Workload        | Throughput (txns/sec) | Latency (ms) |
-| :-------------------- | :-------------------- | :----------- |
-| OLTP_READ_ONLY        | 3276                  | 39           |
-| OLTP_READ_WRITE       | 487                   | 265          |
-| OLTP_WRITE_ONLY       | 1818                  | 70           |
-| OLTP_POINT_SELECT     | 95695                 | 1.3          |
-| OLTP_INSERT           | 6348                  | 20.1         |
-| OLTP_UPDATE_INDEX     | 4052                  | 31           |
-| OLTP_UPDATE_NON_INDEX | 11496                 | 11           |
-| OLTP_DELETE           | 67499                 | 1.9          |
+| Workload              | Throughput(txns/sec) | Latency(ms) |
+| --------------------- | -------------------- | ----------- |
+| oltp_read_only        | 36150                | 1.6         |
+| oltp_read_write       | 2150                 | 11.1        |
+| oltp_multi_insert     | 5450                 | 4.4         |
+| oltp_update_index     | 2420                 | 9.8         |
+| oltp_update_non_index |                      |             |
```

Comment:
I guess numbers are coming here?
What other versions should these changes be copied to?

Reply:
@shantanugupta-yb @shaharuk-yb ^

Reply:
Yes. @shantanugupta-yb  We are trying to maintain benchmark numbers per release going forward.

Reply:
@sudhanshu456 please skip adding ```oltp_update_non_index``` and ```oltp_delete``` for now. Once we get the numbers, we can update the tables with separate diff.